ca_tools/scripts/robot_localization_tf.py

```diff
@@ -0,0 +1,47 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import math
+from geometry_msgs.msg import Twist, Pose, Quaternion
+from tf.transformations import euler_from_quaternion as efq
+from tf.transformations import quaternion_from_euler as qfe
+
+class RobotLocalizationTf():
```

Comment:
Since this class doesn't require to declare an object to use it, consider deleting the constructor and using [static methods](https://www.journaldev.com/18722/python-static-method).

Reply:
Great! Wasn't sure this existed in Python. Thanks